lib/lsperfm/core/run.rb

```diff
@@ -27,7 +27,7 @@ def run(required_events_count, required_run_time, input_lines)
       puts("launching #{command.join(" ")} #{required_events_count} #{required_run_time}") if @debug
       stats = LogStash::PerformanceMeter::Stats.new
       real_events_count = 0
-      Open3.popen3(*@command) do |i, o, e|
+      Open3.popen3({:BUNDLE_BIN_PATH => "", :BUNDLE_GEMFILE => "", :RUBYOPT => ""}, *@command) do |i, o, e|
```

Comment:
I like this change, good idea. Can we make this options passed throught a var? something like

`ENV_OPTS = {:BUNDLE_BIN_PATH => "", :BUNDLE_GEMFILE => "", :RUBYOPT => ""}`

and then have

`Open3.popen3(ENV_OPTS, *@command).... `

what do you think?